include/swift/Runtime/Concurrency.h

```diff
@@ -345,6 +345,11 @@ void swift_continuation_throwingResumeWithError(/* +1 */ SwiftError *error,
                                                 void *continuation,
                                                 const Metadata *resumeType);
 
+/// Drain the queue
+/// If the binary links CoreFoundation, uses CFRunLoopRun
+/// Otherwise it uses dispatchMain.
```

Comment:
Maybe instead of describing impl, we should just note that the function enters an infinite run-loop that drains the main dispatch queue.

Reply:
I do want to note why we don't just call one or the other somewhere in the function. I realize that this isn't exactly why, and more of a what though.

Reply:
Also, maybe given the semantic difference between the two, I think it's somewhat important to note that there is difference based on loading a library.